crates/epaint/src/tessellator.rs

```diff
@@ -2029,8 +2029,8 @@ impl Tessellator {
         use rayon::prelude::*;
 
         // We only parallelize large/slow stuff, because each tessellation job
-        // will allocate a new Mesh, and so it creates a lot of extra memory framentation
-        // and callocations that is only worth it for large shapes.
+        // will allocate a new Mesh, and so it creates a lot of extra memory fragmentation
+        // and call locations that is only worth it for large shapes.
```

Comment:
I think this is supposed to be "allocations".

Reply:
Ah, that makes much more sense